public/app/whiteboards/reuse/whiteboardsReuseController.js

```diff
@@ -111,6 +114,18 @@
     };
 
     /**
+     * Get the assets for the current course through an infinite scroll
+     *
+     * @param  {Asset[]}          assets           All assets from AssetsAPI, including pinned_by_me
+     * @return {Asset[]}                           Filtered set of assets (pinned_by_me are excluded). If user performs search then list is unfiltered.
+     */
+    var notPinnedByMe = $scope.notPinnedByMe = function(assets) {
+      return $scope.isSearch ? assets : _.filter(assets, function(asset) {
+        return !asset.pinned_by_me;
+      });
+    };
```

Comment:
Fyi, the above `notPinnedByMe` will be used in `whiteboards/reuse.html` when filtering assets beyond the "My Pinned" section.